<ui version="4.0" >
 <class>MaskMaker</class>
 <widget class="QDialog" name="MaskMaker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mask Maker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="topLabel" >
       <property name="text" >
        <string>Set criteria for mask generation (criteria passed -> mask = 1)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QDoubleSpinBox" name="minSpinBox" >
         <property name="minimum" >
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ltLabel1" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string> &lt; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="histoLabel" >
         <property name="text" >
          <string>unknown value type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ltLabel2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string> &lt; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxSpinBox" >
         <property name="minimum" >
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaskMaker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaskMaker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
